README.md

```diff
@@ -1,2 +1,63 @@
 # Mergebot
-Mergebot is a GitHub Action that automates merging Dependabot PRs
+
+Mergebot is a GitHub Action that automates merging Dependabot PRs. It supports semantic versioning, enabling you to choose how big of an upgrade bump you allow, and automatically adds reviewers in case the PR requires special attention
+
+Even though it is designed with Dependabot in mind, Mergebot can also be used to automate other GitHub users' PRs. You simply need to override the default input for this
+
+# Dependabot config
+
+Below is a sample dependabot.yml. Notice the schedule is set to daily. This makes the process less conflict-prone. Mergebot can run with any Dependabot config
+
+    version: 2
+    updates:
+      - package-ecosystem: docker
+        directory: /
+        schedule:
+          interval: daily
+      - package-ecosystem: npm
+        directory: /
+        schedule:
+          interval: daily
+      - package-ecosystem: nuget
+        directory: /
+        schedule:
+          interval: daily
+
+# Mergebot config
+
+Below is a sample workflow that uses Mergebot. Notice we recommend checking whether the check_suite was triggered by Dependabot and it finished successfully. This is not necessary, as the action will check this also, but it saves a few seconds from the runtime
+
+Mergebot only supports the check_suite trigger, ensuring it only runs after all the other checks
+
+    name: Dependabot auto merge
+
+    on:
+      check_suite:
+        types:
+          - completed
+
+    jobs:
+      dependabot-auto-merge:
+        if: github.actor == 'dependabot[bot]' && github.event.check_suite.conclusion == 'success'
+        runs-on: ubuntu-latest
+        steps:
+          - uses: e-conomic/mergebot@v0.0.3
+            with:
+              github_token: ${{ secrets.GITHUB_TOKEN }}
+              reviewers: 'reviewer1,reviewer2,reviewer3'
+              team_reviewers: 'team1,team2'
+
+# Mergebot inputs
+
+| Input | Required | Supported options | Default | Note |
+| --- | --- | --- | --- | --- |
+| github_user | no | | dependabot[bot] | Only one value required | 
+| github_token | yes | | | Use ${{ secrets.GITHUB_TOKEN}} or PAT |
+| semver_match | no | patch minor major | patch | Only one value required |
```

Comment:
Can you add the setting to the default config above?
Also, if the user configures major, will that also auto merge patch and minor?

Reply:
Which setting?

Yes, major includes minor, which includes patch

Reply:
include `semver_match` in the sample yaml above

Reply:
It should not be part of the default config, unless the user needs to override it

Reply:
yes, but how does the user override it?

Reply:
It's a standard input, you just need to add it to yml and provide the value

Reply:
Okay, I suggest clarifying that `with` corresponds to the inputs

Reply:
That is the default behavior of GitHub actions. Does not make sense for us to explain how workflows work, IMO